docs/sphinx/themes/globalbftoc.html

```diff
@@ -1,5 +1,5 @@
 <h3><a href="{{ pathto(master_doc) }}">{{ _('Bio-Formats') }}</a></h3>
 {{ toctree()}}
-<a href="http://downloads.openmicroscopy.org/latest/bio-formats5.1/">{{ _('Main Downloads') }}</a></br>
-<a href="http://downloads.openmicroscopy.org/latest/bio-formats-cpp5.1/">{{ _('C++ Downloads') }}</a></br>
+<a href="http://downloads.openmicroscopy.org/latest/bio-formats5.1/">{{ _('Bio-Formats Java Downloads') }}</a></br>
```

Comment:
With the OME Files C++ clarification, this could simply become `Bio-Formats Downloads`. If we agree not to backport, we should be able to point at the latest bio-formats5.2 downloads.